FlowCrypt/Controllers/Setup/SetupViewController.swift

```diff
@@ -104,12 +94,12 @@ extension SetupViewController {
 // MARK: - Key Setup
 
 extension SetupViewController {
-    private func fetchBackupsAndRenderSetupView() {
+    private func fetchBackups() {
```

Comment:
Actually I like the original name :) because when you run this method, several things will happen (by calling other methods) and it feels correct to name it by all the things that will happen.

Maybe more appropriately `fetchBackupsAndRenderResult`